src/bundle/Controller/VersionController.php

```diff
@@ -16,6 +18,32 @@
 
 class VersionController extends Controller
 {
+    public function contentViewTabAction(
```

Comment:
_technically_ it's a location view tab (the view is routed from a location's URL alias).